simple_netconf_client/gui/Dialogs.py

```diff
@@ -12,7 +12,7 @@ def __init__(self, parent, width, height):
         parent.center_dialog(self, width, height)
         about_message = (
             "Simple NETCONF Client\n"
-            f"v{get_version()}\n"
+            f"{get_version()}\n"
```

Comment:
Found the root cause of this.  The previous tag did not have a leading `v`.  I suggest we stick to one scheme in the future for consistency :-)

If you like we can add a `v1.0.0` tag at the same git hash.